src/Microsoft.TestPlatform.ObjectModel/Client/Interfaces/ITestRunEventsHandler2.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Client
+{
+    /// <summary>
+    /// Interface contract for handling test run events during run operation.
+    /// </summary>
+    public interface ITestRunEventsHandler2 : ITestRunEventsHandler
```

Comment:
What could be a better name? 😄

ITestRunEventsHandlerWithAdditionalDebuggingSupport?

cc @AbhitejJohn

Reply:
As per our discussion, the name of this interface shall remain the same.
There's also precedent for this one (see ITestDiscoveryEventsHandler2).